cmd/oci-create-runtime-bundle/main.go

```diff
@@ -91,10 +93,24 @@ It is strongly recommended to keep the default value.`,
 		&v.version, "version", false,
 		`Print version information and exit`,
 	)
+
+	cmd.Flags().StringVar(
+		&v.logLevelString, "log-level", "info",
+		`Log level (panic, fatal, error, warn, info, or debug)`,
+	)
```

Comment:
`--log-level` should be documented in the associated man page (e.g. `oci-create-runtime-bundle.1.md`).